scripts/eherbs.lic

```diff
@@ -1615,6 +1922,35 @@ module EHerbs
       end
     end
 
+    def self.withdraw_note(amount)
+      original_room = Room.current.id
+      Utility.check_cutthroat
+
+      Utility.go2('bank')
+
+      # Can't get a note from Pinefar
+      if XMLData.room_title == '[Pinefar, Depository]'
+        Actions.withdraw if Utility.check_silver < EHerbs.data[:withdraw_amount]
+        EHerbs.data[:need_deposit] = true if EHerbs.data[:deposit_coins]
+        Utility.go2(original_room)
+        return
+      end
+
+      empty_hand
+      result = dothistimeout "withdraw #{amount.to_i} note", 5, /^The teller carefully records the transaction|Very well|The teller hands you the coins|The teller makes some marks on a blank note and hands|The teller makes a few scribblings|The teller taps her quill thoughtfully|seem to have that much/
```

Comment:
This is missing regex for note withdraws from KF, unsure if other towns.
```
The teller purses her lips and says, "Hmm, we will have to give you a salt-stained kraken chit for that amount."  The teller makes some marks on a blank chit and hands it to you, saying "That is a 0 silver surcharge.  Have a nice day."
```

Reply:
Fixed